File: phoebeConverter.py
import json
import os


class PhoebeConverter:

    def __init__(self):
        with open(os.path.join('map', 'map.json'), 'r') as json_data:
            map_list = json.load(json_data)
            self.map = {'ipa': dict(),
                        'xsampa': dict(),
                        'cft': dict()}
            for r in map_list:
                self.map['xsampa'][r[0]] = r[1]
                self.map['ipa'][r[0]] = r[2]
                self.map['cft'][r[0]] = r[3]

    def convert(self, string, system='ipa'):
        if system not in ('ipa', 'xsampa', 'cft'):
            raise ValueError("<system> must be: 'ipa' | 'xsampa' | 'cft'")

        converted = str()

        for i in string:
            converted += self.map[system][i]
        return(converted)


if __name__ == '__main__':
    p = PhoebeConverter()
    phoebe_to_convert = 'Zili bili Ded a bapka'
    converted = p.convert(phoebe_to_convert, system='ipa')
    print('original:  {0}'.format(phoebe_to_convert))
    print('converted: {0}'.format(converted))

File: phoebeConverter.php
<?php

class PhoebeConverter {

	public function __construct() {
		$map_list = json_decode(file_get_contents("map/map.json"), true);
		$this->map = array(
			'ipa'    => array(),
			'xsampa' => array(),
			'cft'    => array(),
		);
		foreach ($map_list as $r){
			$this->map['xsampa'][$r[0]] = $r[1];
			$this->map['ipa'][$r[0]] = $r[2];
			$this->map['cft'][$r[0]] = $r[3];
		}
	}

	public function convert($string, $system){
		if (!in_array($system, array('ipa', 'xsampa', 'cft'))){
			throw new InvalidArgumentException("<system> must be: 'ipa' | 'xsampa' | 'cft'");
		}

		$converted = '';
		$snd_array = str_split($string);

		foreach ($snd_array as $i){
			$converted .= $this->map[$system][$i];
		}
		return $converted;
	}

}

// Example
#$p = new PhoebeConverter();
#$phoebe_to_convert = 'Zili bili Ded a bapka';
#$converted = $p->convert($phoebe_to_convert, 'ipa');
#echo('original: ' . $phoebe_to_convert . ' ||| converted: ' . $converted);
?>
